addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3305,25 +3491,80 @@ var SnippetsMenu = Widget.extend({
                     }
                     $dropZones.droppable({
                         over: function () {
-                            if (dropped) {
+                            if (self.dropped) {
+                                // Checking if the "out" event happened before
+                                // this "over": if `self.dropped` is "true",
+                                // "out" didn't happen because it sets it to
+                                // "false". We are therefore in the case of an
+                                // "over" after an "over" and we need to escape
+                                // the previous dropzone first.
                                 $toInsert.detach();
                                 $toInsert.addClass('oe_snippet_body');
                                 $('.oe_drop_zone').removeClass('invisible');
```

Comment:
Since [this commit](https://github.com/odoo/odoo/commit/31cc10b), the website is inside of an iframe. Due
to that, the snippets menu and snippet editor do not have a direct access to the dropzones. This has been fixed in [this PR](https://github.com/odoo/odoo/pull/136644) as a stable fix.